scripts/openmc-make-depletion-chain-casl

```diff
@@ -0,0 +1,240 @@
+                else:
+                    missing_fpy.append(parent)
+
+        if parent in fpy_data:
+            fpy = fpy_data[parent]
+
+            if fpy.energies is not None:
+                nuclide.yield_energies = fpy.energies
+            else:
+                nuclide.yield_energies = [0.0]
+
+            for E, table_yd, table_yc in zip(nuclide.yield_energies, fpy.independent, fpy.cumulative):
+                yields = defaultdict(float)
+                for product in table_yd:
+                    if product in decay_data:
+                        # identifier
+                        ifpy = CASL_CHAIN[product][2]
+                        # 1 for independent
+                        if ifpy == 1:
+                            if product not in table_yd:
+                                print('No independent fission yields found for {} in {}'.format(product, parent))
+                            else:
+                                yields[product] += table_yd[product].nominal_value
+                        # 2 for cumulative
+                        elif ifpy == 2:
+                            if product not in table_yc:
+                                print('No cumulative fission yields found for {} in {}'.format(product, parent))
+                            else:
+                                yields[product] += table_yc[product].nominal_value
+                        # -1 for stable + unstable
+                        elif ifpy == -1:
+                            if product not in table_yd:
+                                print('No independent fission yields found for {} in {}'.format(product, parent))
+                            else:
+                                yields[product] += table_yc[product].nominal_value
+                                product_meta = '{}_m1'.format(product)
+                                if product_meta in table_yd:
+                                    yields[product] += table_yc[product_meta].nominal_value
+                        # 3 for special treatment with weight fractions
+                        elif ifpy == 3:
+                            for tuple_i in CASL_CHAIN[product][3]:
+                                name_i, weight_i, ifpy_i = tuple_i
+                                if name_i not in table_yd:
+                                    print('No fission yields found for {} in {}'.format(name_i, parent))
+                                else:
+                                    if ifpy_i == 1:
+                                        yields[product] += weight_i * table_yd[name_i].nominal_value
+                                    elif ifpy_i == 2:
+                                        yields[product] += weight_i * table_yc[name_i].nominal_value
+
+                nuclide.yield_data[E] = []
+                for k in sorted(yields, key=openmc.data.zam):
+                    nuclide.yield_data[E].append((k, yields[k]))
+
+    # Display warnings
+    if missing_daughter:
```

Comment:
It doesn't look like `missing_daughter` is used between here and being initialized as empty on line `91`.  Was the intent to append errors to `missing_daughter` (as done for `missing_rx_product` and `missing_fpy`)?